sample-app/src/main/java/com/joinforage/android/example/ui/pos/screens/BalanceInquiryScreen.kt

```diff
@@ -0,0 +1,50 @@
+package com.joinforage.android.example.ui.pos.screens
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+fun BalanceInquiryScreen(
+    onManualEntryButtonClicked: () -> Unit,
+    onSwipeButtonClicked: () -> Unit,
+    onBackButtonClicked: () -> Unit
+) {
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.SpaceBetween,
+        modifier = Modifier.fillMaxSize()
+    ) {
+        Column {
+            Text("How do you want to read your EBT card?")
+            Button(onClick = onManualEntryButtonClicked) {
+                Text("Manually Enter Card Number")
+            }
+            Button(onClick = onSwipeButtonClicked, enabled = false) {
+                Text("Swipe Card")
+            }
+        }
+        Row {
+            Button(onClick = onBackButtonClicked) {
+                Text("Back")
+            }
+        }
+    }
+}
+
+@Preview
+@Composable
+fun BalanceInquiryScreenPreview() {
+    BalanceInquiryScreen(
+        onManualEntryButtonClicked = {},
+        onSwipeButtonClicked = {},
+        onBackButtonClicked = {}
```

Comment:
Yeah! Having this Composable with the `@Preview` decorator will show a preview version of it in the split-view if you choose to use that view mode. Since this `BalanceInquiryScreen` composable needs these arguments, it's easiest to just pass in empty lambdas since the preview's aren't really interactive anyway.